sdk/formrecognizer/Azure.AI.FormRecognizer/api/Azure.AI.FormRecognizer.netstandard2.0.cs

```diff
@@ -437,6 +438,13 @@ public partial class DocumentKeyValuePair
         public Azure.AI.FormRecognizer.DocumentAnalysis.DocumentKeyValueElement Key { get { throw null; } }
         public Azure.AI.FormRecognizer.DocumentAnalysis.DocumentKeyValueElement Value { get { throw null; } }
     }
+    public partial class DocumentLanguage
+    {
+        internal DocumentLanguage() { }
+        public float Confidence { get { throw null; } }
+        public string LanguageCode { get { throw null; } }
```

Comment:
I wondered about whether we should name this property only `Code`. To me it feels like the term "Language Code" is a concept by itself, so I thought it would be better to keep it as it is, even though it could feel a bit redundant:
```cs
foreach (var language in result.Languages)
{
  Console.WriteLine(language.LanguageCode);
}
```
"The language code of the English language" sounds more natural to me than "The code of the English language".

Reply:
I see your point here and I agree. I'll switch Python to use language code.